open-craft/opencraft
# [BB-2220] Add navigation customization page

**Styling of the right side of the navbar needs some tweaks.**

This PR adds button customization page to the new Ocim frontend.

**Testing instructions:**
1. Checkout this branch.
1. Run Ocim (inside vagrant with `make run.dev`).
1. Go to `frontend/` and build the new API client with `./scripts/build-api-client.sh`
1. Run the new frontend: `npm start`.
1. Create a new test instance and log in the console.
1. Go to the `Theme` -> `Navigation` page and make sure it's working as expected (http://localhost:3000/console/theming/navigation).
1. Change colors and check if the values are correctly saved in the backend.

**Reviewers:**
